content/operate/rs/security/access-control/create-users.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: Create users
+alwaysopen: false
+categories:
+- docs
+- operate
+- rs
+description: Assign access control roles to users.
+linkTitle: Create users
+weight: 10
+aliases:
+    - /operate/rs/security/access-control/manage-users/add-users/
+    - /operate/rs/security/access-control/rbac/assign-user-role/
+---
+
+## Add users
+
+To add a user to the cluster:
+
+1. From the **Access Control > Users** tab in the Cluster Manager UI, select **+ Add user**.
+
+    {{<image filename="images/rs/access-control-user-panel.png" alt="Add role with name" >}}
+
+1. Enter the name, email, and password of the new user.
+
+    {{<image filename="images/rs/access-control-user-add.png" alt="Add role with name" >}}
+
+1. Assign a **Role** to the user to grant permissions for cluster management and data access.
+
+    {{<image filename="images/rs/access-control-user-role-select.png" width="300px" alt="Add role to user." >}}
+
+1. Select the **Alerts** the user should receive by email:
+
+    - **Receive alerts for databases** - The alerts that are enabled for the selected databases will be sent to the user. Choose **All databases** or **Customize** to select the individual databases to send alerts for.
```

Comment:
```suggestion
    - **Receive alerts for databases** - The alerts that are enabled for the selected databases will be sent to the user. To send alerts for individual databases, choose **All databases** or **Customize**.
```

Reply:
This suggestion would incorrectly change the meaning. There are two options: either send alerts for all databases by selecting **All databases**, or select **Customize** to send alerts for individual databases.